changes/pr2798.yaml

```
@@ -0,0 +1,5 @@
+enhancement:
+  - "flow.update` accepts the optional `kwarg` `parmerge` that allows flows to be updated with common `Parameters` - [#2501](Can't combine flows that have the same parameter name)"
```

```suggestion
  - "flow.update` accepts the optional `kwarg` `parmerge` that allows flows to be updated with common `Parameters` - [#2501](https://github.com/PrefectHQ/prefect/issues/2501)"
```
FYI these are markdown links